ts-tests/package.json

```
@@ -2,7 +2,7 @@
   "name": "ts-tests",
   "version": "0.0.0",
   "scripts": {
-    "test": "jest"
+    "test": "hardhat compile && jest"
```

For a future PR - maybe we can separate the steps here to make use of turbo's cache

```suggestion
    "compile": "hardhat compile",
    "test": "jest"
```

Then in turbo.json 

```json
"compile": {
  "inputs": ["contracts/*.sol", "hardhat.config.js"],
  "outputs": [ <hardhat compile output dir> ],
},
"test": {
  "inputs": ["__tests__/*.test.ts"],
  "dependsOn": ["compile"]
}
```